README.md

```diff
@@ -52,7 +52,7 @@ See the output of `deepspeech -h` for more information on the use of `deepspeech
 
 ## Prerequisites
 
-* [Python 2.7](https://www.python.org/)
+* [Python 3.5](https://www.python.org/)
```

Comment:
I would change this to Python 3.6, or just leave it as Python.

Reply:
Ok, good idea, I'll do that.